third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -449,11 +449,17 @@ The `authenticator_groups_config` block supports:
 * `security_group` - (Required) The name of the RBAC security group for use with Google security groups in Kubernetes RBAC. Group name must be in format `gke-security-groups@yourdomain.com`.
 
 The `maintenance_policy` block supports:
+* `daily_maintenance_window` - (Required in GA, Optional in Beta) structure documented below.
```

Comment:
Is this still required in GA and optional in Beta? It seems like it functions the same way between beta and GA